tasks/sep/fp-recovery/005-set-game-implementation/justfile

```diff
@@ -36,7 +36,7 @@ prep l1='' l2='':
   CONFIG_URL="https://raw.githubusercontent.com/ethereum-optimism/superchain-registry/refs/heads/main/superchain/configs/{{l1}}/{{l2}}.toml"
   CONFIG_TOML=$(curl -s --show-error --fail "${CONFIG_URL}")
   SYSTEM_CONFIG=$(echo "${CONFIG_TOML}" | yq -p toml .addresses.SystemConfigProxy)
-  OWNER_SAFE=$(echo "${CONFIG_TOML}" | yq -p toml .addresses.ProxyAdminOwner)
+  OWNER_SAFE=$(echo "${CONFIG_TOML}" | yq -p toml .roles.ProxyAdminOwner)
```

Comment:
I don't see this change in the [superchain-registry submodule](https://github.com/ethereum-optimism/superchain-registry/blob/c08331ab44a3645608c08d8c94f78d9be46c13c9/superchain/configs/sepolia/op.toml#L40)

Reply:
It uses the file via a http request to the superchain-registry, not the submodule to ensure it is always using the latest version. That was part of the feedback on the initial template review.